src/base/static/components/app.tsx

```diff
@@ -218,12 +217,14 @@ class App extends React.Component<Props, State> {
           isLoaded: true,
         };
     if (user.isAuthenticated) {
-      Mixpanel.identify(user.id);
-      Mixpanel.track("Successful login");
-      Mixpanel.people.set({
-        name: user.name,
-        username: user.username,
-        id: user.id,
+      import("../utils/mixpanel").then(mixpanel => {
```

Comment:
I don't really see the benefit in doing this, since we are planning to add more telemetry throughout the app. Thus, it seems like a lot more trouble than it's worth, because loading the mixpanel module is going to be inevitable.

If we really don't like mixpanel, then this effort might be better spent looking for a different telemetry library. Google Analytics is one option: https://developers.google.com/analytics/devguides/collection/analyticsjs/creating-trackers

Reply:
Yeah, in testing I'm not sure this really gives us a performance boost anyway, as there seems to always be a mixpanel call when the app loads anyway.

I'm leaning toward undoing this change.